lib/internal/errors.js

```diff
@@ -1425,8 +1425,7 @@ E('ERR_WORKER_PATH', (filename) =>
   'The worker script or module filename must be an absolute path or a ' +
   'relative path starting with \'./\' or \'../\'.' +
   (filename.startsWith('file://') ?
-    ' If you want to pass a file:// URL, you must wrap it around `new URL`.' :
-    ''
+    ' file:// URLs must be wrapped with `new URL`.' : ''
```

Comment:
Nit: Maybe avoid passive voice?

```suggestion
    ' Wrap file:// URLs with `new URL`.' : ''
```